sysl2/sysl/syslutil/typeutil.go

```diff
@@ -0,0 +1,102 @@
+package syslutil
+
+import sysl "github.com/anz-bank/sysl/src/proto"
+
+// GetTypeDetail returns name of the type and details in string format
+func GetTypeDetail(t *sysl.Type) (string, string) {
```

Comment:
nit: Make the return values clearer, either through named return values or a more explicit function name like `GetTypeNameAndDetails`.